tests/integration/pagure/test_generic_commands.py

```diff
@@ -119,11 +119,22 @@ def test_pr_permissions(self):
         assert "lachmanfrantisek" in owners
         assert self.ogr_project.can_merge_pr("lachmanfrantisek")
         project = self.service.get_project(
-            repo=f"playground-{self.service.user.get_username()}",
-            namespace=None,
+            repo="license-validate",
+            namespace="copr",
```

Comment:
I'm not sure how good idea it is to depend on a real project we don't have full control over (meaning if something needs to be changed for whatever reason we can't really object against it).

Reply:
Even though it wouldn't break the tests thanks to recorded requests, it would be a headache to try to re-record it. But maybe I'm overthinking this.

Reply:
good point, I am going to change it